src/frontend/shortlist/src/components/layout/sidebar/SimButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
I don't know where this came from, but this one doesn't have anything to do with the SimDemo stuff; did someone ask to have this included inside your PR?

Reply:
This is not for SimpleView story, this is what Alice worked for side bar and trashcan